sim.py

```diff
@@ -142,10 +142,20 @@ def main():
         '--covenant', help='indicate covenant build for output.', choices=config["covenants"])
     parser.add_argument(
         '--local', help='indicate if the simulation should run local.', action='store_true')
+    parser.add_argument(
+        '--auto_download', help='indicate if we should automatically download latest simc.', action='store_true')
     args = parser.parse_args()
 
     sys.path.insert(0, args.dir)
 
+
+    # Download simc if needed
+    if(args.local and args.auto_download):
```

Comment:
make sure and apply proper pep format here, this should have a space. 
```suggestion
    if (args.local and args.auto_download):
```